packages/launchpad/cypress/e2e/open-mode.cy.ts

```diff
@@ -18,6 +18,34 @@ describe('Launchpad: Open Mode', () => {
     })
   })
 
+  it('opens correctly in unconfigured project with --e2e', () => {
+    cy.scaffoldProject('pristine')
+    cy.openProject('pristine', ['--e2e'])
+    cy.visitLaunchpad()
+    cy.get('h1').should('contain', 'Project Setup')
+  })
+
+  it('opens correctly in unconfigured project with --component', () => {
+    cy.scaffoldProject('pristine')
+    cy.openProject('pristine', ['--component'])
+    cy.visitLaunchpad()
+    cy.get('h1').should('contain', 'Project Setup')
+  })
```

Comment:
I'd expect to find project setup test should live in the ProjectSetup spec and the migration tests in the migration spec file.

Reply:
Good point, not really sure why I chose this one, I think I thought I was in a different file